Makefile

```diff
@@ -45,6 +45,7 @@ purge:
 
 uninstall:
 	$(call UNLINK_LAB_EXTENSION,@elyra/application)
+	$(call UNLINK_LAB_EXTENSION,@elyra/ui-components)
```

Comment:
@ajbozarth should this be **link/unlink** or **install/uninstall** ?

Reply:
link/unlink, like application ui-components is a library not an extension

Reply:
makes sense, I thought it was actually an extension.